build/platform/browser/now.js

```diff
@@ -1,8 +1,24 @@
 'use strict';
 
-var loadNs = performance.now();
-var loadMs = Date.now();
+module.exports = nowMicrosImp();
 
-module.exports = function () {
-  return Date.now();
-};
+// The following snippet was copied directly from the lightstep-tracer-javascript repo.
+// We are currently evaluating lightstep and want to be consistent with how we are gathering
+// our duration metrics.
+// source: https://github.com/lightstep/lightstep-tracer-javascript/blob/c642dd59b4ed13e8832c27345b382f5dbd2a35eb/src/imp/platform/browser/platform_browser.js#L8
+let nowMicrosImp = (function () {
+  // Is a hi-res timer available?
+  if (window.performance &&
+      window.performance.now &&
+      window.performance.timing &&
+      window.performance.timing.navigationStart) {
+      let start = performance.timing.navigationStart;
+      return function () {
+          return Math.floor((start + performance.now()) * 1000.0);
```

Comment:
no `* 1000.0`, since we want ms. the lightstep one gives microsecs

Reply:
Thanks for catching. for some reason I thought lightstep gave nanoseconds.